shipyard_blueprints/airtable/shipyard_airtable/airtable.py

```diff
@@ -28,57 +28,73 @@ def _handle_error(self, error: HTTPError) -> None:
         logger.debug(
             f"Airtable Server Status Code: {error.response.status_code} \n Server Reason: {error.response.reason}"
         )
-        response_details = error.response.json()
-        error_details = response_details.get("error", {})
 
-        if error.response.status_code == 401:
-            raise ExitCodeException(
-                error_details.get(
-                    "message",
-                    "The API key provided is incorrect."
-                    "Please double-check for extra spaces or invalid characters.",
-                ),
-                self.EXIT_CODE_INVALID_CREDENTIALS,
-            )
-        elif error.response.status_code == 403:
-            raise ExitCodeException(
-                error_details.get(
-                    "message",
-                    "The provided key does not have access to the requested resource."
-                    "Please check the scope permissions of the API key.",
-                ),
-                self.EXIT_CODE_INVALID_CREDENTIALS,
-            )
-        elif error_details.get("type") == "TABLE_NOT_FOUND":
-            raise ExitCodeException(
-                error_details.get("message", "The requested table was not found."),
-                self.EXIT_CODE_INVALID_TABLE,
-            )
-        elif error_details.get("type") == "VIEW_NAME_NOT_FOUND":
-            raise ExitCodeException(
-                error_details.get("message", "The requested view was not found."),
-                self.EXIT_CODE_INVALID_VIEW,
-            )
-        elif error_details.get("type") == "UNKNOWN_FIELD_NAME":
-            raise ExitCodeException(
-                error_details.get("message", "Unknown field name."),
-                self.EXIT_CODE_BAD_REQUEST,
-            )
-        elif error.response.status_code == 404:
-            raise ExitCodeException(
-                error_details.get("message", "The requested resource was not found."),
-                self.EXIT_CODE_RESOURCE_NOT_FOUND,
-            )
-        elif error.response.status_code == 429:
-            raise ExitCodeException(
-                error_details.get(
-                    "message", "The rate limit for the API key has been exceeded."
-                ),
-                self.EXIT_CODE_RATE_LIMIT,
-            )
-        else:
+        try:
+            response_details = error.response.json()
+
+            error_details = response_details.get("error", {})
+            if isinstance(error_details, str):
```

Comment:
There are scenarios where the server returns {"error":"error_reason"} instead of the more common 
```{'error': {'message': 'Invalid permissions, or the requested model was not found. Check that both your user and your token have the required permissions, and that the model names and/or ids are correct.', 'type': 'INVALID_PERMISSIONS_OR_MODEL_NOT_FOUND'}}```

This was causing a `'str' object has no attribute 'get'`